package.json

```diff
@@ -95,7 +95,9 @@
     "swiper": "^8.3.2",
     "tailwind-merge": "^1.3.0",
     "tailwindcss": "^3.1.4",
-    "yaml": "^2.2.2"
+    "yaml": "^2.2.2",
+    "fs": "^0.0.1-security",
+    "path": "^0.12.7"
```

Comment:
```suggestion
```

These packages are included by default in node js. Kindly don't add it in `package.json`.

Reply:
Thanks, changed

Reply:
> Kindly share the workflow test links you did in fork repository. 

@akshatnema  - https://github.com/AtharvChandratre/website/actions/runs/6218086013

Reply:
You haven't pushed your changes 😃

Reply:
Yep, just did! Was looking into why https://github.com/AtharvChandratre/website/actions/runs/6365025644/job/17281733745?pr=3 fails intermittently. Seems to be because it's calling the YouTube API, which seems to not respond sometimes.